schema.json

```diff
@@ -0,0 +1,496 @@
+                                    "minLength": 1
+                                },
+                                "description": {
+                                    "type": "string"
+                                },
+                                "website": {
+                                    "type": "string"
+                                },
+                                "source_code_url": {
+                                    "type": "string",
+                                    "minLength": 1
+                                },
+                                "support_url": {
+                                    "type": "string"
+                                },
+                                "published_at": {
+                                    "type": "string",
+                                    "format": "date-time"
+                                },
+                                "source": {
+                                    "type": "object",
+                                    "required": [
+                                        "git",
+                                        "commit"
+                                    ],
+                                    "additionalProperties": false,
+                                    "properties": {
+                                        "git": {
+                                            "type": "string"
+                                        },
+                                        "commit": {
+                                            "type": "string"
+                                        }
+                                    }
+                                },
+                                "asset_urls": {
+                                    "type": "object",
+                                    "additionalProperties": false,
+                                    "maxProperties": 2,
+                                    "properties": {
+                                        "icon.svg": {
+                                            "type": "string"
+                                        },
+                                        "icon.png": {
+                                            "type": "string"
+                                        }
+                                    }
+                                },
+                                "host_os_tags": {
+                                    "type": "array",
+                                    "items": {
+                                        "type": "string"
+                                    }
+                                },
+                                "project_type_tags": {
+                                    "type": "array",
+                                    "uniqueItems": true,
+                                    "items": {
+                                        "type": "string",
+                                        "enum": [
```

Comment:
enum doesn't contain empty, however that is also valid, pls check

Reply:
in stepman models, this field has omitempty qualifier, so it will not present in the JSON if has no value and it is not required in the schema: https://github.com/bitrise-io/bitrise-steplib/pull/2431/files#diff-76d5ffeac090c76ed5f4c581f02f3c53R132

the only issue would be if someone defines:

```
project_type_tags:
- ""
```

which i think should not be allowed.